### DISCUSSION TITLE
EMF support on Linux through libuemf

### DISCUSSION THREAD

**Discussion by gaby64** [ImageMagick/ImageMagick#6680] [Contribute]
### Is your feature request related to a problem? Please describe.

EMF has been Windows only for far too long.

### Describe the solution you'd like

allow use of libuemf like libwmf is used for WMF

### Describe alternatives you've considered

emf2svg

### Additional context

EMF use is widespread with MS Word use.

**Comment by dlemstra** [comment]
Anyone is free to submit a pull request that offers support for this but I would advise to use a dependency that has a non GPL license to be compatible with our Apache license.

**Comment by ashahabov** [comment]
EMF is still not supported for Linux?
